src/style.rs

```diff
@@ -114,37 +146,37 @@ pub struct TabStyle {
     /// Colour of the tab's background. By `Default` it's [`Color32::WHITE`]
     pub bg_fill: Color32,
 
-    /// Color of tab title when an inactive tab is unfocused.
-    pub text_color_unfocused: Color32,
-
-    /// Color of tab title when an inactive tab is focused.
-    pub text_color_focused: Color32,
+    /// Color of the title text.
+    pub text_color: Color32,
+}
 
-    /// Color of tab title when an active tab is unfocused.
-    pub text_color_active_unfocused: Color32,
+/// Specifies the look and feel of the tab body.
+#[derive(Clone, Debug)]
+pub struct TabBodyStyle {
+    /// Inner margin of tab body. By `Default` it's `Margin::same(4.0)`
+    pub inner_margin: Margin,
 
-    /// Color of tab title when an active tab is focused.
-    pub text_color_active_focused: Color32,
+    /// Color of the tabs border. By `Default` it's ['Stroke::default']
```

Comment:
Contrary to what the comment says, this is no longer just colour – it's now a `Stroke`.